### ISSUE TITLE
add art galleries

### ISSUE THREAD

**Issue by IndyHurt** [tilezen/vector-datasource#990]
* **What did you see?**
I did not see art galleries
* **What did you expect to see?**
Art galleries
* **What map location are you having problems with?** _City and country are helpful, as well as tile coordinates or latitude / longitude._  
Here are several from overpass:  [California Galleries](http://overpass-turbo.eu/map.html?Q=%2F*%0AThis%20query%20looks%20for%20nodes%2C%20ways%20and%20relations%20%0Awith%20the%20given%20key%2Fvalue%20combination.%0AChoose%20your%20region%20and%20hit%20the%20Run%20button%20above!%0A*%2F%0A%5Bout%3Ajson%5D%5Btimeout%3A25%5D%3B%0A%2F%2F%20gather%20results%0A(%0A%20%20%2F%2F%20query%20part%20for%3A%20“tourism%3Dgallery”%0A%20%20node%5B%22tourism%22%3D%22gallery%22%5D(31.05293398570514%2C-127.35351562499999%2C43.29320031385282%2C-111.24755859375)%3B%0A%20%20way%5B%22tourism%22%3D%22gallery%22%5D(31.05293398570514%2C-127.35351562499999%2C43.29320031385282%2C-111.24755859375)%3B%0A%20%20relation%5B%22tourism%22%3D%22gallery%22%5D(31.05293398570514%2C-127.35351562499999%2C43.29320031385282%2C-111.24755859375)%3B%0A)%3B%0A%2F%2F%20print%20results%0Aout%20body%3B%0A%3E%3B%0Aout%20skel%20qt%3B)
* **Screenshot?** 

One expected at Minna & 2nd: 
<img width="307" alt="overpass example" src="https://cloud.githubusercontent.com/assets/959783/17940297/55e80032-69e3-11e6-8fb0-bfe477a80458.png">
<img width="307" alt="missing in zinc example" src="https://cloud.githubusercontent.com/assets/959783/17940600/b6f85808-69e4-11e6-8fa5-75bc9eeca70f.png">

Link to map above:  [Targeted Editing - Collections Map](https://mapzen-data.github.io/targeted-editing/te-collections/map/#19/37.78735/-122.39957)

* **OpenStreetMap Tags** - link to queries if helpful: [OpenStreetMap gallery queries](https://github.com/mapzen-data/targeted-editing/blob/gh-pages/queries/collections_sql.sql#L4)
  - Identified appropriate tag
    - [`tourism=gallery`](http://wiki.openstreetmap.org/wiki/Tag:tourism%3Dgallery)

  - Variants found with tag<b>info</b>
    - `amenity=gallery`
    - `amenity=art_gallery`
    - `amenity=arts_gallery`
    - `amenity=contemporary_art_gallery`
    - `building=gallery`
    - `gallery=*`
    - `tourism=art_gallery`
    - `tourism=Art Gallery`
    - `tourism=Art_gallery`
    - `museum=gallery`

**Comment by nvkelso** [comment]
**tl;dr**

Only add:

- `tourism=gallery` as a new **kind:gallery**
- This should vary in size like museum for features that are tagged on area features (like buildings), between **15** and **17** depending on size and defaulting to **17** if it's a node without area. 
- Indy's **node** example is: http://www.openstreetmap.org/node/2026996113, it should appear by default at zoom **17** (not 16 as seen in the museum example below).
- Example **area** feature: [Royal Academy of Arts](http://www.openstreetmap.org/way/31510288#map=15/51.5092/-0.1289) in London with expected min_zoom at z15. 

Museum example in [pois.yaml](https://github.com/tilezen/vector-datasource/blob/master/yaml/pois.yaml#L114-L116):

```
  - filter: {tourism: museum}
    min_zoom: LEAST(zoom + 1.43, 16)
    output: {kind: museum}
```

_The other tags aren't supported in the wiki or by common usage in the data, we should ignore them._

===

**Approximate uses of each of these via TagInfo:**

  - Identified appropriate tag
    - [`tourism=gallery`](http://wiki.openstreetmap.org/wiki/Tag:tourism%3Dgallery) - 2138

  - Variants found with tag<b>info</b>
    - `amenity=gallery` - 87
    - `amenity=art_gallery` - 47
    - `amenity=arts_gallery` - 1
    - `amenity=contemporary_art_gallery` - 4
    - `building=gallery` - 10
    - `gallery=*` - 25
    - `tourism=art_gallery` - 5
    - `tourism=Art Gallery` - 1
    - `tourism=Art_gallery` - 1
    - `museum=gallery` - 8

**Comment by nvkelso** [comment]
Verified!
